exonum-java-binding-cryptocurrency-demo/frontend/src/plugins/blockchain.js

```diff
@@ -128,7 +128,11 @@ module.exports = {
         }
         const body = CreateTransactionProtobuf.encode(data).finish()
 
-        return axios.post(TX_URL, buildMessage(header, body, keyPair))
+        return axios.post(TX_URL, buildMessage(header, body, keyPair), {
```

Comment:
Would you please extract this duplicated code in a method `submitTransaction(message)`?

Reply:
I fix it